vignettes/vignette_titanic.Rmd

```diff
@@ -21,11 +21,11 @@ knitr::opts_chunk$set(
 # Data for Titanic survival
 
 Let's see an example for `DALEX` package for classification models for the survival problem for Titanic dataset.
-Here we are using a dataset `titanic` avaliable in the `DALEX` package. Note that this data was copied from the `stablelearner` package.
+Here we are using a dataset `titanic_imputed` avaliable in the `DALEX` package.
```

Comment:
let's keep here an information about the source (stablelearner) of the data.

Reply:
Information kept.